docs/release-notes/8.12.asciidoc

```diff
@@ -22,6 +22,23 @@ Select the **Edit the query filter using DSL** option.
 ====
 // end::known-issue-173958[]
 
+// tag::known-issue-174844[]
+[discrete]
+.Unrelated property differences in prebuilt rule update comparison
+[%collapsible]
+====
+*Details* +
+The JSON comparison for updated prebuilt detection rules might display some properties used for internal processing, which doesn't accurately indicate how the rule will change if you update it.
+
+For example, if you added automated actions or an exception list to an installed rule, the comparison shows the JSON properties `actions`, `response_actions`, or `exceptions_list` in the **Base version** (your installed version) but not in the **Update** column (Elastic's latest version). When you update the rule, it will still include your actions or exceptions — they will not be removed. 
+
+Similarly, the comparison might show a difference in the `enabled` property, but upgrading the rule will not change whether your installed rule is enabled or not. Other properties that might display in the comparison but don't actually indicate rule configuration changes include `execution_summary`, `timestamp_override_fallback_disabled`, `meta`, `filters`, `updated_at`, `output_index`, and `note`.
```

Comment:
please remove filters, timestamp_override_fallback_disabled and note from this list - those are relevant properties.

Reply:
```suggestion
Similarly, the comparison might show a difference in the `enabled` property, but upgrading the rule will not change whether your installed rule is enabled or not. Other properties that might display in the comparison but don't actually indicate rule configuration changes include `execution_summary`, `timestamp_override_fallback_disabled`, `meta`, `filters`, `updated_at`, and `output_index`.
```